site/content/contribute/mobile/push-notifications/android.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Android Push Notifications"
+date: 2015-05-20T11:35:32-04:00
+weight: 1
+subsection: "Setup Push Notifications"
+---
+
```

Comment:
In this doc, please help add periods after each sentence for consistency. 

Also, pictures are not previewing on this page and not sure if they are on other pages either.

Reply:
images are showing

![image](https://user-images.githubusercontent.com/6757047/54537247-f78eb380-4970-11e9-8c82-527f28a8e002.png)

probably you need to see this with hugo and not the github previewer